docs/guide/openclaw/index.md

```
@@ -0,0 +1,179 @@
+
+This view shows usage grouped by individual OpenClaw sessions. Session IDs come from the JSONL filename stem (the part before `.jsonl`, ignoring `.deleted.<ts>` or `.reset.<ts>` suffixes), and metadata records the last activity date and provider that produced the most recent activity.
+
+```bash
+ccusage openclaw session
+ccusage openclaw session --json
+ccusage openclaw session --since 2026-05-09
+```
+
+### Example Output
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)